docs/site/Controllers.md

```diff
@@ -196,6 +196,49 @@ export class HelloController {
 - `@param.query.number` specifies in the spec being generated that the route
   takes a parameter via query which will be a number.
 
+## Modifying Specifications Created by Controller Generator
+
+You can run controller generator to generate REST controllers with CRUD methods.
+The command and prompts are explained in page
+[controller-generator](./Controller-generator.md#rest-controller-with-crud-methods).
+To modify the OpenAPI specifications of REST controllers, you can leverage the
+[specification enhancers](Extending-OpenAPI-specification.md).
+
+For example, the default naming convention for a path's `operationId` is
+`${controllerName}.${methodName}`. To override the `operationId` with a custom
+one `${controllerName}-${methodName}`, you can define an enhancer as:
+
+```ts
+import {bind} from '@loopback/core';
+import {
+  mergeOpenAPISpec,
+  asSpecEnhancer,
+  OASEnhancer,
+  OpenApiSpec,
+} from '@loopback/rest';
+
+/**
+ * A spec enhancer to modify `operationId` in paths
+ */
+@bind(asSpecEnhancer)
+export class OperationSpecEnhancer implements OASEnhancer {
+  name = 'operationIdEnhancer';
+  // takes in the current spec, modifies it, and returns a new one
+  modifySpec(spec: OpenApiSpec): OpenApiSpec {
+    const paths = spec.paths;
+    for (const path in paths) {
+      for (const op in path) {
+        const operationId = paths[path][op].operationId;
+        // change operationId from 'MyController.MyMethod' to
+        // 'MyController-MyMethod'
+        if (operationId) operationId.replace('.', '-');
```

Comment:
The updated value needs to be assigned to `paths[path][op].operationId`

Reply:
good catch.